README.md

```diff
@@ -68,6 +68,20 @@ Kindev will be installed in the `.kind` folder.
 Vscode handles all port-forwarding automagically, so kindev and all endpoints will be available all the same from the host.
 Simply point `KUBECONFIG` to `.kind/.kind/kind-config` and use `kubectl` as usual.
 
+> [!NOTE]
+> On linux, Podman and rootless Docker have been found to cause issues.  
+> Ensure that you are using **rootful Docker** to save yourself loads of trouble.  
+>
+> Additionally, on linux, you might encounter issues with `iptables`.  
```

Comment:
Some of the part is rather specific to Fedora. Probably best to mention that some of the stuff is specific to that.
E.g On Arch the `iptables` package is already the "legacy" one.
Afaik on Ubuntu the package is also called `iptables-legacy`.
There isn't a need to specify it for all flavors, just mention that the example you specified is specific to Fedora